### DISCUSSION TITLE
An error occurred when navigating to a page with await inside

### DISCUSSION THREAD

**Discussion by ilyaDegtyarenko** [nuxt/nuxt#23221] [Questions]
The error occurs when I go to the third level child page, then back to the second and then go to the page with await inside

Reproduction:
[https://stackblitz.com/edit/nuxt-starter-btlcfb?file=pages%2Fwaiting.vue](https://stackblitz.com/edit/nuxt-starter-btlcfb?file=pages%2Fwaiting.vue)

Instructions for reproducing the bug:
1. INDEX PAGE must be initial
2. Go to SETTING/PROJECT PAGE
3. Go to SETTING/PROJECT/CREATE PAGE
4. Go to SETTING/PROJECT PAGE
5. Go to WAITING PAGE
6. Look in console (error due to waiting)

Errors in console:
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'namespaceURI')
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'exposed')
<img width="647" alt="image" src="https://github.com/nuxt/nuxt/assets/37239202/6a747fe0-a7cd-41a4-abf7-e2efcd99868b">
<img width="648" alt="image" src="https://github.com/nuxt/nuxt/assets/37239202/d9a84a21-2b5c-429e-bd81-f26c24f88c75">

This error was not fixed for more than a month, maybe several

**Comment by ilyaDegtyarenko** [comment]
@danielroe can you pay attention, please?

**Reply by akereachu** [reply]
@ilyaDegtyarenko I had a similar issue removing page-key solved it for me, Strange!
`<NuxtPage page-key="static" /> to <NuxtPage  />`

**Reply by ilyaDegtyarenko** [reply]
If you look in the reproduction, I don't have a page-key there

**Comment by alexovn** [comment]
Same error 😕

**Comment by TheAlexLichter** [comment]
Let's track in https://github.com/nuxt/nuxt/issues/24991
